openhcl/underhill_attestation/src/lib.rs

```diff
@@ -366,8 +366,10 @@ pub async fn initialize_platform_security(
 
     // Check if the VM id has been changed since last boot with KP write
     let vm_id_changed = if key_protector_by_id.found_id {
+        tracing::info!("VM Id has changed since last boot");
```

Comment:
id might not change. We have log for this boolean below

Reply:
I think this is a more explicit message.  I will leave here for now unless it is really taking up too much space.

Reply:
message can be misleading because the boolean value is based on the statement below

Reply:
I see, I misread your comment.